integrationtests/org.eclipse.xtext.swtbot.tests/pom.xml

```diff
@@ -17,10 +17,10 @@
 			<plugin>
 				<groupId>org.eclipse.tycho</groupId>
 				<artifactId>tycho-surefire-plugin</artifactId>
+				<version>1.3.0</version>

```

Comment:
why not use tycho-version variable?

Reply:
please manage the plugin version in the parent pom.xml in pluginManagement.